modules/meta.analysis/R/approx.posterior.R

```diff
@@ -43,7 +43,9 @@ approx.posterior <- function(trait.mcmc, priors, trait.data=NULL, outdir=NULL, f
     
     dat    <- trait.mcmc[[trait]]
     vname  <- colnames(dat[[1]])
-    dat    <- as.matrix(dat)[,'beta.o']
+    if('beta.o' %in% vname) {
```

Comment:
Let's not assume that beta.o is the name but work to generalize away from that legacy assumption. Let's discuss whether we have to assume that the object is a mcmc.list or not in the context of the more general PDA design and asynchronous updating. In general I think it's not going to be the case that we'll want that as a general assumption.